charts/fluid/fluid/templates/crd-upgrade.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Could you add upgrade folder, and put crd-upgrade.yaml inside it.

Reply:
Done. Moved to `templates/upgrade/crd-upgrade.yaml`